plugin/gocode/list.go

```diff
@@ -55,11 +56,29 @@ func newSuggestionList(driver gxui.Driver, theme *basic.Theme, proj setting.Proj
 		editor:  editor,
 		ctrl:    ctrl,
 		applier: applier,
+		gocode:  gocode,
 	}
 
 	s.Init(s, theme)
 	s.OnGainedFocus(s.Redraw)
 	s.OnLostFocus(s.Redraw)
+	s.OnKeyPress(func(ev gxui.KeyboardEvent) {
+		if ev.Key == gxui.KeyEnter {
+			s.driver.CallSync(func() {
+				s.gocode.Confirm(s.editor)
+			})
+		} else if ev.Key == gxui.KeyEscape {
+			s.driver.CallSync(func() {
+				s.gocode.Cancel(s.editor)
+			})
+		}
+	})
+	s.OnDoubleClick(func(gxui.MouseEvent) {
+		s.driver.CallSync(func() {
+			s.gocode.Confirm(s.editor)
+		})
+	})
+
```

Comment:
This works for now, but in the future, I'd like to modify it so that it is a hook on the `input.Handler`.